components/core/src/clp_s/SchemaWriter.hpp

```diff
@@ -40,19 +40,17 @@ class SchemaWriter {
     /**
      * Stores the schema to disk.
      */
-    void store();
+    void store(ZstdCompressor& compressor);
 
     /**
      * Closes the schema writer.
      * @return the compressed size of the schema table in bytes
      */
     [[nodiscard]] size_t close();
 
+    uint64_t get_num_messages() const { return m_num_messages; }
```

Comment:
Missing docstring.

Reply:
I remembered we don't need docstrings for basic getters/setters

Reply:
Fair. Feel free to ignore this then.